app-network/frrouting/autobuild/defines

```diff
@@ -0,0 +1,34 @@
+PKGNAME=frrouting
+PKGSEC=net
+PKGDEP="c-ares libcap json-c libyang ncurses net-snmp openssl readline iproute2 logrotate lsof shadow"
+BUILDDEP="flex bison python-3"
+PKGDES="IP routing protocol suite (BGP, OSPF, IS-IS, RIP, EIGRP, BABEL, PIM, NHRP, PBR)"
+
+AUTOTOOLS_AFTER="--sysconfdir=/etc/frr \
+                 --localstatedir=/var/lib/frr \
```

Comment:
Are you sure? This parameter is usually used to set a prefix, not a full path ("$LOCALSTATEDIR"/lib/frr).